engine/build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
     [group: 'org.ow2.asm', name: 'asm-tree', version: '5.0.4'],
 	     
     // Flatbuffers
-    [group: 'com.github.davidmoten', name: 'flatbuffers-java', version: '1.4.0.1'],
+    [group: 'com.google.flatbuffers', name: 'flatbuffers-java', version: '1.11.0'],
```

Comment:
@tmightyquinn does this change do anything bad to releases / affect the scaffold?

Reply:
I think this is fine for scaffold as we build the jar on release; competitors don't need to compile this. However, have you checked that the new version is compatible with what the client is running? Need to ensure that whatever the engine writes to the wire is consumable by the client.

Reply:
yup, it's still compatible! (apparently flatbuffers is always forwards/backwards compatible, so there should never be an issue with engine/client communication. the reason I upgraded this, though, is because the flatbuffers runtime version for engine was really old, and that _does_ need to match the version of the compiler you use to make the Java header files, which is why I always had to ask wflms to compile the schema for me :o)